app/__init__.py

```diff
@@ -43,9 +44,11 @@ def create_app(config_class=Config):
     health = Healthcheck(app)
     health.add_check(FlaskRunningChecker())
 
-    # instantiate email to LA and place and fund types mapping used for authorizing submissions
-    app.config["EMAIL_TO_LA_AND_PLACE_NAMES_AND_FUND_TYPES"] = copy(EMAIL_DOMAIN_TO_LA_AND_PLACE_NAMES_AND_FUND_TYPES)
-    app.config["EMAIL_TO_LA_AND_PLACE_NAMES_AND_FUND_TYPES"].update(app.config.get("ADDITIONAL_EMAIL_LOOKUPS", {}))
+    # TODO: TOWNS_FUND_AUTH is currently stored in const.py but this isn't isn't a good solution.
```

Comment:
Could a database table contain these auth mappings in future? Just a thought

Reply:
Yes - I think that's where we would go with this. We'd ideally want the funds to manage their auth info via some sort of FE that does CRUD operations